python/docs/source/conf.py

```diff
@@ -93,7 +93,7 @@ def setup(app):
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'default'
+html_theme = 'sphinx_rtd_theme'
```

Comment:
@jackgerrits  Build Documentation if failing due to this

```
Theme error:
sphinx_rtd_theme is no longer a hard dependency since version 1.4.0. Please install it manually.(pip install sphinx_rtd_theme)
make: *** [html] Error 2
Makefile:19: recipe for target 'html' failed
```
So can we install it in the build?

Reply:
I have added it to the build.